<launch>
<arg name="robot_id" value="$(optenv ROBOT_ID unnamed_robot)"/>
<!-- Follow_Line -->
  <group ns="$(arg robot_id)">
    <node pkg="pileec2223_control_methods" type="Follow_Line"
        name="Follow_Line" clear_params="true"
        output="$(optenv ROBOT_OUTPUT screen)">
      <rosparam command="load" subst_value="True"
          file="$(find pileec2223_control_methods)/config/Follow_Line.yaml"/>
    </node>
  </group>
</launch>
